.travis.yml

```diff
@@ -28,8 +28,8 @@ before_install:
 before_script:
 - bundle install
 - yarn install
-- RAILS_ENV=test bundle exec rake db:create db:migrate
-- RAILS_ENV=test bundle exec rake assets:precompile
+- RAILS_ENV=test DATABASE_URL=postgresql://postgres@localhost:5433 bundle exec rake db:create db:migrate
+- RAILS_ENV=test DATABASE_URL=postgresql://postgres@localhost:5433 bundle exec rake assets:precompile
```

Comment:
can this go in a env piece?
eg.

```
env:
  global:
    DATABASE_URL="postgresql://postgres@localhost:5433"
```

Reply:
Probably, I'll move it once it's working...